src/Components/Genre/Genre.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import classes from "./Genre.module.css"
+
+class Genre extends React.PureComponent {
+  render() {
+    const genres = this.props.genres.map((item, index) => <li key={index}>{item}</li>)
```

Comment:
It is not recommended to use index as a key, more on this: https://reactjs.org/docs/lists-and-keys.html#keys